apps/preview/next/pages/showcases/Drawer/TransitionAndCloseButton.tsx

```diff
@@ -1,70 +1,105 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import { ShowcasePageLayout } from '../../showcases';
 // #region source
-import { SfDrawer, SfButton, SfIconFavorite, SfIconClose, useTrapFocus } from '@storefront-ui/react';
+import {
+  SfDrawer,
+  SfButton,
+  SfRadio,
+  SfIconFavorite,
+  SfIconClose,
+  useTrapFocus,
+  SfDrawerProps,
+} from '@storefront-ui/react';
 import { useState, useRef } from 'react';
-import { CSSTransition } from 'react-transition-group';
+import { Transition } from 'react-transition-group';
+import classNames from 'classnames';
 
 export default function DrawerWithTransition() {
   const [open, setOpen] = useState(false);
+  const [checkedPlacement, setCheckedPlacement] = useState<SfDrawerProps['placement']>('left');
```

Comment:
Could you rename it to just `placement`? I think it'd be easier to scan the code later to find which parts relates to 'left' and which to 'right' variant. Also, there's an enum `SfDrawerPlacement` that you can use to define the type here.

```suggestion
  const [placement, setPlacement] = useState<`${SfDrawerPlacement}`>('left');
```